osu.Game/Overlays/OverlayRulesetTabItem.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.UserInterface;
+using osu.Framework.Input.Events;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Rulesets;
+using osuTK.Graphics;
+using osuTK;
+using System;
+
+namespace osu.Game.Overlays
+{
+    public class OverlayRulesetTabItem : TabItem<RulesetInfo>, IHasAccentColour
+    {
+        protected readonly OsuSpriteText Text;
+        private readonly FillFlowContainer content;
+
+        public override bool PropagatePositionalInputSubTree => Enabled.Value && !Active.Value && base.PropagatePositionalInputSubTree;
+
+        private Color4 accentColour;
+
+        public Color4 AccentColour
+        {
+            get => accentColour;
+            set
+            {
+                if (accentColour == value)
+                    return;
+
+                accentColour = value;
+
+                updateState();
+            }
+        }
```

Comment:
This seems to be essentially identical to:
```csharp
private Bindable<Color4> accentColour;
public Color4 AccentColour
{
    get => accentColour.Value;
    set => accentColour.Value = value;
}
```
and then having a `BindValueChanged` in `LoadComplete`. I know you just moved it from its previous location but if we're touching this anyway then we might as well remove the reinvented wheel.